Documentation/cloud-config.md

```diff
@@ -4,6 +4,7 @@
 
 **NOTE**: coreos-cloudinit is no longer under active development and has been superseded by [Ignition][ignition]. For more information about the recommended tools for provisioning Container Linux, refer to the [provisioning documentation][provisioning].
 
+[ignition]: https://coreos.com/ignition/docs/latest/
```

Comment:
I'm not a fan of linking to our site from GitHub. Does it make sense to link directly to the Ignition repo instead?